### PR TITLE
regionmask v0.4.0

### PR DESCRIPTION
**Pull request by mathause**

### CODE REVIEW

**Comment by mathause** [comment]
apologies, added it again - do I need to start over or is this fine?

File: recipes/regionmask/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (CJ-Wright) [inline recipes/regionmask/meta.yaml]: Please remove these comments

File: recipes/regionmask/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - matplotlib
+    - shapely
+    - six
+    - cartopy
+    - geopandas
+
+test:
+  imports:
+    - regionmask
+  requires:
```
Comment (CJ-Wright) [inline recipes/regionmask/meta.yaml]: If you don't need to run the tests then you don't need the `requires` section.

File: recipes/regionmask/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - numpy
+    - matplotlib
+    - shapely
+    - six
+    - cartopy
+    - geopandas
+
+test:
+  imports:
+    - regionmask
```
Comment (CJ-Wright) [inline recipes/regionmask/meta.yaml]: Would it be possible to get a few more import test?

**Comment by CJ-Wright** [comment]
In the future please don't PR from master into master, please make your own branch on your fork.

**Pull request merged**